packages/botonic-pulumi/src/aws/rest-server.ts

```diff
@@ -172,7 +188,7 @@ export class RestServer extends AWSComponentResource<RestServerArgs> {
         },
         { ...opts, parent: this }
       )
-      const url = pulumi.interpolate`${deployment.invokeUrl}${API_PATH_NAME}`
+      const url = pulumi.interpolate`${deployment.invokeUrl}${API_PATH_NAME}/`
       this.url = url
```

Comment:
The url can be assigned directly to `this.url`

Reply:
agree 👍